src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
  * реалізації ArrayList (default capacity, newCapacity...)</p>
  */
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private Object[] elements;
+    private int size;
+    private int index;
```

Comment:
do you need both of these variables?

Reply:
Actually, we don't.
Made it for better code understanding, forgot to remove.
Fixed